src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <a
+        href="#"
+        class="card__link"
+        data-qa="hover"
+        >
+        <img
+          src="images/imac.jpeg"
+          class="card__img"
+          alt="APPLE A1419 iMac">
+      </a>
```

Comment:
Pay attention on indents - it should be 2 spaces

link with data-qa="hover" should be a BUY button, not an image
```suggestion
    <div class="card" data-qa="card">
      <img
        src="images/imac.jpeg"
        class="card__img"
        alt="APPLE A1419 iMac"
      >
```